### DISCUSSION TITLE
Hide Arrow Button

### DISCUSSION THREAD

**Discussion by rimzzlabs** [MurhafSousli/ngx-gallery#443] [Q&A] [answered]
The gallery works perfectly fine! however I want to remove the arrow button, any idea how?
I've been looking through the gallery API, and it seems there is no props that allow me to hide the arrow

**Comment by MurhafSousli** [comment, marked as answer]
@rizkimcitra What arrow button? if you mean the navigation buttons, you can remove them by disabling the input `[nav]`

```html
<gallery [nav]="false"></gallery>
```

https://github.com/MurhafSousli/ngx-gallery/wiki/Gallery-API

**Reply by rimzzlabs** [reply]
sorry for late reply, I mean this one 
![image](https://user-images.githubusercontent.com/62492410/180035142-06be3f0d-7cbe-4046-96f0-3aed49bab70d.png)

**Reply by MurhafSousli** [reply]
Yes, the same answer! check it here https://stackblitz.com/edit/ngx-gallery-uws8zu?file=src%2Fapp%2Fhome%2Fhome.component.html

**Reply by rimzzlabs** [reply]
thank you sir, all set now😄🎉
